### ISSUE TITLE
extension:install bug when using discover install

### ISSUE THREAD

**Issue by myser90** [joomlatools/joomlatools-console#61]
In Extension.Install.php
```
 $model = new \InstallerModelDiscover();
 $model->discover();

 $results = $model->getItems();
```

But $model->getItems() return limited result. By default only 20 extensions have been returning. 

In getItems:
```
		try
		{
			// Load the list items and add the items to the internal cache.
			$this->cache[$store] = $this->_getList($this->_getListQuery(), $this->getStart(), $this->getState('list.limit'));
		}
```

As a result, if the extension is not included in the first twenty, it will be ignored.

**Comment by stevenrombauts** [comment]
Thanks @myser90, this should be easily solved by setting the limit state to a larger number. Pull Requests welcome! :)

**Comment by stevenrombauts** [comment]
This has been solved and will be included in the upcoming 1.5 release!
